src/python/README.md

```diff
@@ -104,20 +103,20 @@ This code snippet is provided for reference only as Cedar does
 not yet support local environments.
 
 ```sh
-./genny run -o metrics.csv -m cedar-csv InsertRemove.yml
```

Comment:
Could we add a comment somewhere saying that the metrics format is defined in the workload YAML files (see HelloWorld.yml)? David seemed confused about it last Thursday.